charts/argo-cd/templates/_helpers.tpl

```diff
@@ -241,7 +241,7 @@ NOTE: Configuration keys must be stored as dict because YAML treats dot as separ
 {{- if .Values.commitServer.enabled -}}
 {{- $_ := set $presets "commit.server" (printf "%s:%s" (include "argo-cd.commitServer.fullname" .) (.Values.commitServer.service.port | toString)) -}}
 {{- end -}}
-{{- range $component := tuple "applicationsetcontroller" "controller" "server" "reposerver" "notificationscontroller" "dexserver" -}}
+{{- range $component := tuple "applicationsetcontroller" "controller" "server" "reposerver" "notificationscontroller" "dexserver" "commitserver" -}}
```

Comment:
I wonder if we'd need to match the case here - it's defined as `commitServer` :thinking:

Reply:
I think we don't need this.
According to this https://argo-cd.readthedocs.io/en/stable/operator-manual/server-commands/additional-configuration-method/ and this https://github.com/argoproj/argo-cd/blob/d2b881ae4adb0cb46c78feab490ff3177740fe11/docs/operator-manual/argocd-cmd-params-cm.yaml#L221C1-L223C33